onnxruntime/test/perftest/performance_runner.cc

```diff
@@ -109,6 +109,7 @@ void PerformanceResult::DumpToFile(const std::basic_string<ORTCHAR_T>& path, boo
 }
 
 Status PerformanceRunner::Run() {
+  ZoneScopedN("PerformanceRunner::Run");
```

Comment:
Should this be inside an `#ifdef`?

Do we really need OpenCL specific profiling in this shared code though? We already have multiple ways to profile things, so it would be best to avoid adding one more that is EP specific here.